docs/lib/DEBUGGING.md

```diff
@@ -121,9 +121,9 @@ The uVisor code is instrumented to output debug information when it is relevant.
 
 Instead, debug events and messages are forwarded to a special unprivileged box, called a *debug box*. A debug box is configured just like any other secure box, but it registers with uVisor to handle debug callbacks. These callbacks must adhere to a format that uVisor provides, in the form of a debug box driver.
 
-The debug box driver is encoded in a standard table (a C `struct`) that a debug box must populate at initialization time. A debug box can decide to implement only some of the available handlers though they must all exist at least as empty functions. Otherwise, the program behavior might be unpredictable.
+The debug box driver is encoded in a standard table (a C `struct`) that a debug box must populate statically. A debug box can decide to implement only some of the available handlers though they must all exist at least as empty functions. Otherwise, the program behavior might be unpredictable.
```

Comment:
Originally this document was VERY specific, with code examples.
But, latest PRs removed the code examples, referring the reader to the the debug-fault example.